### PR TITLE
Set up React App for Maths App application.

### PR DESCRIPTION
**Pull request by TaiwoEnoch**
# I create this PR to feature the following

- [ ] Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config). Remember that this is your first React project and it requires a special linters setup. Check the instructions in the linters-config README.
- [ ] Make sure that you used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- [ ] Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

- [ ] First, make sure you have [Node.js](https://nodejs.org/en/) installed on your machine
- [ ] The easiest way to create a React application is using [Create React App](https://create-react-app.dev/)
- [ ] The application should run in the browser without errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,161 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (m05e5) [inline README.md:161]: - [ ] Great job with your professional readme file. currently, the license link is broken kindly add the License file provided by microverse to your app

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @TaiwoEnoch , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights

- Clean and well-organized code
- professional readme file 
- no linters error

## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by didierganthier — APPROVED** [review]
Hi @TaiwoEnoch ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitFlow ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Comment by TaiwoEnoch** [comment]
Thanks for your time @didierganthier

**Pull request merged**